katpoint/timestamp.py

```diff
@@ -135,7 +135,7 @@ def __iadd__(self, other):
 
     def __isub__(self, other):
         """Subtract seconds (as floating-point number) from timestamp in-place."""
-        self.secs -= float(other)
+        self.secs -= other
```

Comment:
This is preferable, yes, because `Timestamp -= Timestamp` would be bad (the end result has to be a `Timestamp`).